cdap-ui/app/cdap/components/TextboxOnValium/index.js

```diff
@@ -32,8 +32,14 @@ export default class TextboxOnValium extends Component {
     }
   }
   updateTextValue(e) {
+    let textValue = e.target.value;
+
+    if (!this.props.allowSpace) {
+      textValue = textValue.trim();
```

Comment:
So do we only need to remove spaces from beginning and end of the string? Spaces in the middle are ok?